comfy/utils.py

```diff
@@ -6,6 +6,14 @@
 import numpy as np
 from PIL import Image
 import logging
+import json
+
+
+def dump_json(data):
+    # Roundtrip once to get rid of NaN, Infinity and -Infinity
+    good_data = json.loads(json.dumps(data), parse_constant=lambda x: None)
+    return json.dumps(good_data, allow_nan=False)
```

Comment:
Is this really the best way of doing this?

Reply:
I took a look at how JSONEncoder is implemented and there seemed to be no easy way to override its handling of floats without basically re-implementing the entire thing. This is the least awful way I found to solve the problem.

An alternative would be to use simplejson instead, but that introduces a new dependency.